From d23f47d15d42252a84f010d53ffe1cd9b232cbca Mon Sep 17 00:00:00 2001
From: Daan De Meyer <daan.j.demeyer@gmail.com>
Date: Fri, 24 Jan 2025 10:54:51 +0100
Subject: [PATCH] mkosi: Drop usage of _systemd_QUIET in arch build script

We dropped the variable in the packaging specs for Arch to keep the
integration points as minimal as possible so let's stop using it in
the build script as well.

(cherry picked from commit 8dab59e610720c022f962311069c95fd8569f82a)
---
 mkosi.images/build/mkosi.conf.d/arch/mkosi.build.chroot | 1 -
 1 file changed, 1 deletion(-)

diff --git a/mkosi.images/build/mkosi.conf.d/arch/mkosi.build.chroot b/mkosi.images/build/mkosi.conf.d/arch/mkosi.build.chroot
index 202646f0ad..2a5b10251b 100755
--- a/mkosi.images/build/mkosi.conf.d/arch/mkosi.build.chroot
+++ b/mkosi.images/build/mkosi.conf.d/arch/mkosi.build.chroot
@@ -85,7 +85,6 @@ env --chdir="pkg/$PKG_SUBDIR" \
     $( ((WITH_TESTS)) || echo --nocheck) \
     --force \
     _systemd_UPSTREAM=1 \
-    _systemd_QUIET=$( ((MESON_VERBOSE)); echo $? ) \
     BUILDDIR="$PWD/pkg/$PKG_SUBDIR" \
     PKGDEST="$BUILDDIR" \
     PKGEXT=".pkg.tar" \
